lnmuxrpc/lnmux.proto

```diff
@@ -73,3 +77,23 @@ message CancelInvoiceRequest {
 }
 
 message CancelInvoiceResponse {}
+
+message ListInvoiceRequest {
+    uint64 num_max_invoices = 1;
+    uint64 index_offset = 2;
+}
+
+message ListInvoiceResponse {
+    repeated Invoice invoice = 1;
+    uint64 first_index_offset = 2;
+    uint64 last_index_offset = 3;
```

Comment:
This can be derived via `index_offset + len(invoice)?`

I am not sure if we should add these convenience fields or not. An api like this has probably been defined thousands of times in the wider world, but I have no idea what's the common way to do it.

Reply:
From what I see using
```
 uint64 first_index_offset = 2;
 uint64 last_index_offset = 3;
```

Is pretty common in the wider world

Imagine we did some manual intervention in the DB: we canceled/deleted invoices with sequence_number == 1 and 3

If I request:

REQUEST( index_off_set=1, max_invoices=3)

I will get [2,4,5]

index_offset != first_index_offset
and
last_index_offset != index_offset + len(invoices)

Reply:
I see. Ok, let's keep it indeed.